docs/getting-started.md

```diff
@@ -109,22 +109,26 @@ WIREMOCK_BASEURL="http://localhost:8080"
 ```bash
 cd DfE.FindInformationAcademiesTrusts.UnitTests/playwright
 
-# run docker image with build flag to watch for the latest code changes
-docker compose -f ../../docker/docker-compose.ci.yml up -d --build
-
 # install dependencies
 npm install
 
+# run docker image with build flag to watch for the latest code changes
```

Comment:
We should reword this comment. The build flag doesn't watch for changes, it just forces a rebuild (rather than using whatever docker already has cached)